examples/DAPDialect/RFFT.cpp

```diff
@@ -0,0 +1,75 @@
+//===- WhisperPreprocessor.cpp --------------------------------------------===//
```

Comment:
`//===- WhisperPreprocessor.cpp --------------------------------------------===//`change to
`//===- RFFT.cpp - Example of DAP RFFT -------------------------------------===//`